### PR TITLE
Simian Uniform Update

### PR DESCRIPTION
**Pull request by lmenvs**
# About The Pull Request
Adds all missing uniform sprites to Simians.  That's right, each clothing article will now shrink fit to your Simian coworkers.  That's weird.  Whatever.

Also ended up cleaning some of the files up in our modular folder for consistency, hence large amount of files changed.

## Why It's Good For The Game
Properly fitting clothing will boost morale and work ethic.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog

:cl:
add: Simians now have clothing sprites for every wearable item in the game.
/:cl:

### CODE REVIEW

**Comment by Oricana-16** [comment]
🙏🙏🙏 simians will be balling

**Comment by lmenvs** [comment]
i broke jumpsuits somehow, i will fix when i get back

File: code/__HELPERS/icons.dm [hunk 1]
```diff
@@ -1195,87 +1181,13 @@ GLOBAL_LIST_INIT(freon_color_matrix, list("#2E5E69", "#60A2A8", "#A1AFB1", rgb(0
...
- * Arguments:
- * * thing - either a /icon object, or an object that has an appearance (atom, image, mutable_appearance).
- * * target - either a reference to or a list of references to /client's or mobs with clients
- * * icon_state - string to force a particular icon_state for the icon to be used
- * * dir - dir number to force a particular direction for the icon to be used
- * * frame - what frame of the icon_state's animation for the icon being used
- * * moving - whether or not to use a moving state for the given icon
- * * sourceonly - if TRUE, only generate the asset and send back the asset url, instead of tags that display the icon to players
- * * extra_clases - string of extra css classes to use when returning the icon string
- */
```
Comment (nednaZ) [inline code/__HELPERS/icons.dm:1271]: If these comments are still relevant and accurate, I'd like them to still be there.

File: code/__HELPERS/icons.dm [hunk 2]
```diff
@@ -1195,87 +1181,13 @@ GLOBAL_LIST_INIT(freon_color_matrix, list("#2E5E69", "#60A2A8", "#A1AFB1", rgb(0
...
-
-/proc/icon2html(atom/thing, client/target, icon_state, dir = SOUTH, frame = 1, moving = FALSE, sourceonly = FALSE)
+/proc/icon2html(thing, target, icon_state, dir = SOUTH, frame = 1, moving = FALSE, sourceonly = FALSE)
 	if (!thing)
 		return
 
 	var/key
-	var/icon/icon2collapse = thing
+	var/icon/I = thing
```
Comment (nednaZ) [inline code/__HELPERS/icons.dm:1190]: 1 letter var

File: code/__HELPERS/icons.dm [hunk 3]
```diff
@@ -1300,41 +1209,37 @@ GLOBAL_LIST_INIT(freon_color_matrix, list("#2E5E69", "#60A2A8", "#A1AFB1", rgb(0
 			if(sourceonly)
 				return SSassets.transport.get_asset_url(name)
 			return "<img class='icon icon-misc' src='[SSassets.transport.get_asset_url(name)]'>"
-		//its either an atom, image, or mutable_appearance, we want its icon var
-		icon2collapse = thing.icon
+		var/atom/A = thing
```
Comment (nednaZ) [inline code/__HELPERS/icons.dm:1212]: 1 letter var

**Review by nednaZ — CHANGES_REQUESTED** [review]
It's fully functional, though I'm not a huge fan of how it's finding the modular items.
Seems hacky, though half of SS13 code seems that way.
Going to testmerge

File: code/__HELPERS/icons.dm (hunk 2 above)
Comment (polygoblyn) [reply code/__HELPERS/icons.dm:1190]: Uhhh I never touched this, I will revert changes on this file

**Comment by nednaZ** [comment]
No issues found during testmerge

**Pull request merged**